getsolus/packages
# playonlinux: Housekeeping

**Summary**

- Install appstream metadata (Part of https://github.com/getsolus/packages/issues/1389)
- Added homepage key (Part of https://github.com/getsolus/packages/issues/411)
- Use patched upstream desktop file
- Use patched upstream Makefile

**Test Plan**

- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`
- Verify homepage loads relevant information
- Make sure app icon switch properly depending on icon theme (Tested on GNOME, Budgie, Plasma)
- Install `foobar2000` on Budgie VM

**Checklist**
